src/internal/got/types.ts

```diff
@@ -85,6 +85,10 @@ export interface GetUserLookupResponse {
     cftools_id: string,
 }
 
+export interface GetPlayerLookupResponseWithNotice extends GetUserLookupResponse {
+    notice: string;
```

Comment:
just put the `notice` field into the `GetUserLookupResponse` type as an optional field :)

Reply:
Sure thing!

Reply:
Fixed in `f1d0ae77a9cc81f82ddd0e06f4fb4d5ba22e6fdd`